.github/workflows/medcat-v2-tutorials_main.yml

```diff
@@ -4,8 +4,11 @@ on:
   push:
     branches: [ main ]
   pull_request:
-    branches: [ main ]
-
+    paths:
+      - 'medcat-v2/**'
```

Comment:
Just as a comment, I don't think this _currently_ has much effect since the tutorials target a hard coded version rather than the one in the PR. But in principle that is a separate issue we should address.